source/includes/administration/_organizations.md

```diff
@@ -974,4 +974,80 @@ curl "https://cloudmc_endpoint/api/v1/organizations/87895f43-51c1-43cc-b987-7e30
 
 Required | &nbsp;
 ---- | ----
-`billingEmails`<br/>*map* | A map of billing emails and names associated with them. There is a maximum of three emails that can be configured. Each entry must have a valid email specified along with a name.
\ No newline at end of file
+`billingEmails`<br/>*map* | A map of billing emails and names associated with them. There is a maximum of three emails that can be configured. Each entry must have a valid email specified along with a name.
+
+<!-------------------- FIND BILLING CYCLE -------------------->
+### Get a billing cycle for an org
+
+`GET /organizations/:organization_id/billing_cycles/:billing_cycle_id`
+
+Retrieves the billing cycle for the organization.
+
+```shell
+# Retrieve billing cycle
+curl "https://cloudmc_endpoint/api/v1/organizations/c01e2bd4-50c4-4ef4-b756-f728823309a4/billing_cycles/8259182d-5234-4a78-adf6-7edc11db2e3b" \
+   -H "MC-Api-Key: your_api_key"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": {
+    "pricingPackages": [
+      {
+        "endDate": "2021-12-01",
+        "packageId": "508a1f5f-2504-41ac-b48f-b87e757840a4",
+        "name": {
+          "en": "Cox Pricing Package"
+        },
+        "startDate": "2021-11-01"
+      }
+    ],
+    "deferred": false,
+    "invoices": [],
+    "discounts": [
+      {
+        "endDate": "2021-11-11",
+        "name": {
+          "en": "Credit 01"
+        },
+        "discountId": "e6ada5f6-9806-466c-8522-314da152a8e7",
+        "type": "CREDIT",
+        "startDate": "2021-11-01",
+        "remaining": {
+          "discountedProducts": {},
+          "discountedCategories": {},
+          "packageDiscount": 10.0,
+          "labels": {}
+        }
+      },
+    ],
+    "endDate": "2021-12-01",
+    "currency": "USD",
+    "id": "8259182d-5234-4a78-adf6-7edc11db2e3b",
+    "state": "IN_PROGRESS",
+    "startDate": "2021-11-01"
+  }
+}
+```
+
+Attributes | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | The id of the billing cycle.
+`startDate`<br/>*string* | The start date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) of the billing cyle.
+`endDate`<br/>*string* | The end date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) of the billing cyle.
+`currency`<br/>*string* | The currency associated to the organization.
+`deferred`<br/>*boolean* | Indicated whether the billing cycle is deferred or not.
+`invoices`<br/>*Array[[Invoice](#administration-invoices)]* | List of invoices associated with the billing cycle.
```

Comment:
For the type, I think we generally use Array[Object]

Reply:
are we returning the complete invoice object or just the id?

Reply:
Just IDs

Reply:
I guess I can change it to Array[UUID] in this case

Reply:
4f8cda86ce0fcb9055e399d1e28e37c7502d4cd9